src/workerd/io/frankenvalue.h

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) 2024 Cloudflare, Inc.
+// Licensed under the Apache 2.0 license found in the LICENSE file or at:
+//     https://opensource.org/licenses/Apache-2.0
+
+#pragma once
+
+#include <workerd/io/frankenvalue.capnp.h>
+#include <workerd/jsg/jsg.h>
+
+namespace workerd {
+
+class Frankenvalue {
```

Comment:
A code comment description of what this is and how/why to use it would be helpful.